src/pages/compose/index.tsx

```diff
@@ -58,23 +65,17 @@ const Compose = () => {
     setDescription(projectData.description);
   }, [projectData.name, projectData.description]);
 
+
   const handleSaveProjectInfo = async () => {
     setSaveModalVisible(false);
-    showSuccessToast("Saved Successfully");
-    try {
-      const response = await axios.post(
-        "http://localhost:8080/sourcecourse/project",
-        {
-          name: name,
-          description: description,
-        }
-      );
-      if (response.data !== -1) {
-        setProjectId(response.data.uid);
-        handleIconClick("HddFilled");
-      }
-    } catch (error) {
-      console.error(error);
+    //dispatch done to post compose page name and desc to db
+    const success = await dispatch(postComposeNameDescRequest(name, description));
+    if (success) {
+      showSuccessToast("Saved Successfully");
```

Comment:
Messages and hardcoded values needs to be moved to constant file

Reply:
Done